eyeshade/workers/reports.js

```diff
@@ -1386,7 +1430,7 @@ exports.workers = {
         })
         if ((!summaryP) && (!payload.owner)) data.push([])
         data.push({
-          publisher: 'TOTAL',
+          note: 'TOTAL OUT',
```

Comment:
ditto the comment above, though i suspect "OUT" will be replaced with "SETTLEMENTS?